src/ff/galois_field.rs

```diff
@@ -201,22 +249,9 @@ macro_rules! bit_array_impl {
                 }
             }
 
-            impl TryFrom<u128> for $name {
-                type Error = String;
-
-                /// Fallible conversion from `u128` to this data type. The input value must
-                /// be at most `Self::BITS` long. That is, the integer value must be less than
-                /// or equal to `2^Self::BITS`, or it will return an error.
-                fn try_from(v: u128) -> Result<Self, Self::Error> {
-                    if u128::BITS - v.leading_zeros() <= Self::BITS {
-                        Ok(Self::truncate_from(v))
-                    } else {
-                        Err(format!(
-                            "Bit array size {} is too small to hold the value {}.",
-                            Self::BITS,
-                            v
-                        ))
-                    }
```

Comment:
As commented above, this is removed to resolve a conflict, but I will bring this back in the next PR.